recipes/libtiff/all/conanfile.py

```diff
@@ -149,6 +149,14 @@ def _configure_cmake(self):
             if self._has_webp_option:
                 self._cmake.definitions["webp"] = self.options.webp
             self._cmake.definitions["cxx"] = self.options.cxx
+
+            # Workaround for cross-build to at least iOS/tvOS/watchOS,
+            # when dependencies like libdeflate, jbig and zstd are found with find_path() and find_library()
+            # see https://github.com/conan-io/conan-center-index/issues/6637
+            if tools.cross_building(self):
```

Comment:
one more thing to support in build helper, which is unlikely to be addressed...

Reply:
https://github.com/conan-io/conan/issues/9202#issuecomment-894076226